versions.json

```diff
@@ -1,9 +1,9 @@
 {
   "nixpkgs": {
-    "owner": "NixOS",
+    "owner": "mkg20001",
```

Comment:
We need to clean this up before merging!

Reply:
upstream PR for this is https://github.com/NixOS/nixpkgs/pull/137661